shared/domains/chelonia/chelonia.js

```diff
@@ -91,6 +163,15 @@ sbp('sbp/selectors/register', {
       return stack
     }
   },
+  'chelonia/with-env': async function (contractID: string, env: Object, sbpInvocation: Array<*>) {
+    const savedEnv = this.env
+    this.env = env
+    try {
+      return await sbp('okTurtles.eventQueue/queueEvent', `chelonia/env/${contractID}`, sbpInvocation)
```

Comment:
Why is this using a new event queue `chelonia/env/` ?

Reply:
We picked up discussion of the solution to this issue in a later thread here: https://github.com/okTurtles/group-income/pull/1257#pullrequestreview-1155587120